apps/ewallet_config/lib/ewallet_config/validator.ex

```diff
@@ -0,0 +1,170 @@
+defmodule EWalletConfig.Validator do
```

Comment:
If it's part of `EWalletDB`, then we can't use those validation functions in `EWalletConfig`. `EWalletConfig` has no dependencies on any other app, but all other apps depend on it directly or indirectly. I could move back some of the functions to `EWalletDB` tho, since they are not used in the validations in `EWalletConfig` anymore.

Reply:
Moving back would be nice I think. I feel `EWalletDB` having to depend on validators from `EWalletConfig` is a bit counterintuitive.

Also, if the `EWalletConfig`'s validators are the same as `EWalletDB`'s, having duplicate code in this is case fine?